dev/reference/error-codes.md

```diff
@@ -18,13 +18,102 @@ TiDB 兼容 MySQL 的错误码，在大多数情况下，返回和 MySQL 一样
 | 8003 | `ADMIN CHECK TABLE` 命令在遇到行数据跟索引不一致的时候返回该错误 |
 | 8004 | 单个事务过大，原因及解决方法请参考[这里](/dev/faq/tidb.md#433-transaction-too-large-是什么原因怎么解决) |
 | 8005 | 事务在 TiDB 中遇到了写入冲突，原因及解决方法请参考[这里](/dev/faq/tidb.md#九故障排除) |
+| 8018 | 插件无法重新载入，原因是没有载入过 |
+| 8019 | 重新载入的插件版本与之前不同，无法重新载入 |
+| 8020 | 表被锁 |
+| 8021 | key 不存在 |
+| 8022 | 事务提交失败，但可以进行重试 |
+| 8023 | 不能设置空值 |
+| 8024 | 非法的事务 |
+| 8025 | 写入的单条键值对过大 |
+| 8026 | 没有实现的接口 |
+| 8027 | 表结构版本过期 |
+| 8028 | 表结构发生了变化 |
+| 8029 | 错误值 |
+| 8030 | 转变为带符号正整数发生了越界，显示为负数 |
+| 8031 | 负数转变为无符号数转变为正数 |
```

Comment:
```suggestion
| 8031 | 负数转变为无符号数，然后再转变为正数 |
```

Reply:
I think it changed the original mean.

Reply:
@jackysp Then could you make it clearer and easier to understand?